OpenRA.Game/Translation.cs

```diff
@@ -53,63 +63,65 @@ static IEnumerable<MessageContext> GetMessageContext(string language, string[] t
 				var stream = fileSystem.Open(path);
 				using (var reader = new StreamReader(stream))
 				{
-					var options = new MessageContextOptions { UseIsolating = false };
-					var messageContext = new MessageContext(language, options);
-					var errors = messageContext.AddMessages(reader);
+					var parser = new LinguiniParser(reader);
+					var resource = parser.Parse();
+					var errors = resource.Errors;
 					foreach (var error in errors)
 						Log.Write("debug", error.ToString());
 
-					yield return messageContext;
+					bundle.AddResourceOverriding(resource);
```

Comment:
This name seems to imply that later files override earlier ones. Won't this mean that the english strings will always override other languages?